spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindAssertionsWithSideEffects.java

```diff
@@ -0,0 +1,63 @@
+package edu.umd.cs.findbugs.detect;
+
+import org.apache.bcel.Const;
+import edu.umd.cs.findbugs.BugInstance;
+import edu.umd.cs.findbugs.BugReporter;
+import edu.umd.cs.findbugs.ba.SignatureParser;
+import edu.umd.cs.findbugs.util.MutableClasses;
+
+/**
+ * This detector can find assertions that violates the EXP06 rule.
```

Comment:
assertions that violates -> violate

Reply:
This typo was fixed in 7bacf835615e80642aa22b27d932c191e8ff6338, so this can be resolved.